control-plane/csi-driver/src/context.rs

```diff
@@ -57,6 +57,8 @@ pub enum Parameters {
     PoolTopologyAffinity,
     #[strum(serialize = "poolTopologySpread")]
     PoolTopologySpread,
+    #[strum(serialize = "quiesce")]
+    Quiesce,
```

Comment:
I'm wondering if this should be two fold, example quiesceFs and quiesceBlock?

Reply:
Or maybe just add a Block Variant to the Quiesce? That we can extend once we decide to do sync for block?

Reply:
yeah perhaps